v21.2/authentication.md

```diff
@@ -68,6 +68,10 @@ CockroachDB offers the following methods for client authentication:
 
    Note that the client still needs the CA certificate to validate the nodes' certificates.
 
+   {{site.data.alerts.callout_success}}
+   <span class="version-tag">New in v21.2</span>: For improved performance, CockroachDB securely caches password authentication information for users. To limit the authentication latency of users logging into a new session, we recommend that you run bulk `ROLE` operations ([`CREATE ROLE`](create-role.html), [`ALTER ROLE`](alter-role.html), [`DROP ROLE`](drop-role.html)) inside of a transaction., and run any regularly-scheduled `ROLE` operations together, rather than at different times throughout the day.
```

Comment:
```suggestion
   <span class="version-tag">New in v21.2</span>: For improved performance, CockroachDB securely caches password authentication information for users. To limit the authentication latency of users logging into a new session, we recommend that you run bulk `ROLE` operations ([`CREATE ROLE`](create-role.html), [`ALTER ROLE`](alter-role.html), [`DROP ROLE`](drop-role.html)) inside a transaction, and run any regularly-scheduled `ROLE` operations together, rather than at different times throughout the day.
```

Reply:
Done.